src/ontology/mondo-edit.obo

```diff
@@ -464825,6 +464835,487 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
+xref: OMIM:620359 {source="MONDO:equivalentTo"}
+is_a: MONDO:0014471 {source="OMIM:620359"} ! mitochondrial proton-transporting ATP synthase complex deficiency
+
+[Term]
+id: MONDO:0957260
+name: combined low LDL and fibrinogen
+xref: OMIM:620364 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620364"} ! hereditary disease
+
+[Term]
+id: MONDO:0957261
+name: pulmonary fibrosis and/or bone marrow failure syndrome, telomere-related, 7
+xref: OMIM:620365 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000148 {source="OMIM:620365"} ! pulmonary fibrosis and/or bone marrow failure, telomere-related
+
+[Term]
+id: MONDO:0957262
+name: osteopetrosis, autosomal recessive 9
+xref: OMIM:620366 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019026 {source="OMIM:620366"} ! autosomal recessive osteopetrosis
+
+[Term]
+id: MONDO:0957263
+name: pulmonary fibrosis and/or bone marrow failure syndrome, telomere-related, 8
+xref: OMIM:620367 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000148 {source="OMIM:620367"} ! pulmonary fibrosis and/or bone marrow failure, telomere-related
+
+[Term]
+id: MONDO:0957264
+name: cerebroretinal microangiopathy with calcifications and cysts 3
+xref: OMIM:620368 {source="MONDO:equivalentTo"}
+is_a: MONDO:0012815 {source="OMIM:620368"} ! Coats plus syndrome
+
+[Term]
+id: MONDO:0957265
+name: congenital myopathy 22B, severe fetal
+xref: OMIM:620369 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019952 {source="OMIM:620369"} ! congenital myopathy
+
+[Term]
+id: MONDO:0957266
+name: RECON progeroid syndrome
+xref: OMIM:620370 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620370"} ! hereditary disease
+
+[Term]
+id: MONDO:0957267
+name: neurodevelopmental disorder with intracranial hemorrhage, seizures, and spasticity
+xref: OMIM:620371 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620371", source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
+is_a: MONDO:0700092 {source="https://orcid.org/0000-0001-5208-3432"} ! neurodevelopmental disorder
+
+[Term]
+id: MONDO:0957268
+name: hypersulfaturia
+xref: OMIM:620372 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620372"} ! hereditary disease
+
+[Term]
+id: MONDO:0957270
+name: muscular dystrophy, limb-girdle, autosomal recessive 28
+xref: OMIM:620375 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015152 {source="OMIM:620375"} ! autosomal recessive limb-girdle muscular dystrophy
+
+[Term]
+id: MONDO:0957271
+name: autoinflammatory disease, systemic, with vasculitis
+xref: OMIM:620376 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620376", source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
+is_a: MONDO:0019751 {source="https://orcid.org/0000-0001-5208-3432"} ! autoinflammatory syndrome
+
+[Term]
+id: MONDO:0957273
+name: Charcot-Marie-Tooth disease, dominant intermediate A
+xref: OMIM:620378 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620378"} ! hereditary disease
```

Comment:
Could the parent be 
MONDO:0019548
'autosomal dominant intermediate Charcot-Marie-Tooth disease'

Reply:
done